src/collections/approaches/approaches.11tydata.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
-const generatePermalink = require("../../utils/generatePermalink.js");
-const getLang = require("../../utils/getLang.js");
+const generatePermalink = require("../../_utils/generatePermalink.js");
```

Comment:
This can be restructured to match the [directory data files](https://github.com/fluid-project/trivet/blob/main/src/collections/pages/pages.11tydata.js#L3) in Trivet. This will also necessate adding [`en-CA.11tydata.js`](https://github.com/fluid-project/trivet/blob/main/src/collections/pages/en-CA/en-CA.11tydata.js) files in the specific locale directory.